acceptance/helper.rb

```diff
@@ -744,123 +726,43 @@ def object_diff(obj1, obj2)
   # End Object diff functions
   ##############################################################################
 
-  def install_puppet_from_package
+  def install_puppet
     os_families = test_config[:os_families]
     hosts.each do |host|
       os = os_families[host.name]
 
       case os
       when :debian
-        if options[:type] == 'aio' then
-          on host, "apt-get install -y puppet-agent"
-          on host, "apt-get install -y puppetserver"
-        else
-          on host, "apt-get install -y puppet puppetmaster-common"
-        end
-      # Puppet 3.7.4 is broken on fedora, pinning to 3.7.3 until it's fixed
-      when :fedora
-        on host, "yum install -y puppet-3.7.3"
+        on host, "apt-get install -y puppet-agent"
       when :redhat
-        if options[:type] == 'aio' then
-          on host, "yum install -y java-1.7.0-openjdk"
-          on host, "yum install -y puppet-agent"
-          on host, "yum install -y puppetserver"
-        else
-          on host, "yum install -y puppet"
-        end
+        on host, "yum install -y puppet-agent"
       else
         raise ArgumentError, "Unsupported OS '#{os}'"
       end
     end
   end
 
-  # This helper has been grabbed from beaker, and overriden with the opts
-  # component so I can add a new 'refspec' functionality to allow a custom
-  # refspec if required.
-  #
-  # Once this methodology is confirmed we should merge it back upstream.
-  def install_from_git(host, path, repository, opts = {})
-    name   = repository[:name]
-    repo   = repository[:path]
-    rev    = repository[:rev]
-
-    target = "#{path}/#{name}"
-
-    step "Clone #{repo} if needed" do
-      on host, "test -d #{path} || mkdir -p #{path}"
-      on host, "test -d #{target} || git clone #{repo} #{target}"
-    end
-
-    step "Update #{name} and check out revision #{rev}" do
-      commands = ["cd #{target}",
-                  "remote rm origin",
-                  "remote add origin #{repo}",
-                  "fetch origin #{opts[:refspec]}",
-                  "clean -fdx",
-                  "checkout -f #{rev}"]
-      on host, commands.join(" && git ")
+  def munge_etc_hosts_to_disable_dujour_checkins
+    databases.each do |database|
+      stub_hosts_on(database, 'updates.puppetlabs.com' => '127.0.0.1')
```

Comment:
@nwolfe Here's the example of the `stub_hosts_on` vs. `apply_manifest_on` for dealing with the acceptance test pre_suite business for dujour.